scripts/db_run.sh

```diff
@@ -45,6 +45,30 @@ get_postgres_server_port() {
     echo "$(sudo netstat -lptn | grep "^tcp " | grep postgres | sed 's/\s\+/ /g' | cut -d ' ' -f4 | cut -d ':' -f2)"
 }
 
+start_postgres_daemon() {
+    PGUSER=postgres
+    PGDATA="/var/lib/postgres/data"
+    PGLOG="$PGDATA/serverlog"
+
+    if [ ! -d "$PGDATA" ]; then
+        mkdir $PGDATA
+        chown -R $PGUSER:$PGUSER $PGDATA
+    fi
+
+    INIT_COMMAND="initdb --locale en_US.UTF-8 -E UTF8 -D '$PGDATA'"
+    su $PGUSER -c "$INIT_COMMAND"
+
+    echo "[+] Starting PostgreSQL ..." 1>&2
+    START_COMMAND="postgres -D '$PGDATA' &"
+    su $PGUSER -c "$START_COMMAND" >>$PGLOG 2>$PGLOG
+
+    #Check the logfile for errors.
+    status=$(cat $PGLOG | grep "fail" | wc -l)
```

Comment:
The output gets appended to log right? So any failed old runs might trigger this condition?